modules/sdk/node-build/cssLoad/writeCSSLoaderJavaScriptModules.mjs

```diff
@@ -0,0 +1,33 @@
+import fs from 'fs/promises';
+import path from 'path';
+
+import getFlatName from "../util/getFlatName.mjs";
+
+export default async function writeCSSLoaderJavaScriptModules(projectExports, projectWebContextPath) {
+	return Promise.all( 
```

Comment:
Do we still need to `await` here? or is it fine because it is returning a promise

Reply:
You meant `async`, right? (not `await`)

You can use `async` even if you don't use `await` in any place as per the spec and I think it makes a lot of sense to add it since it tells the reader what to expect from the function.